packages/@unimodules/core/android/build.gradle

```diff
@@ -67,4 +67,5 @@ android {
 
 dependencies {
   implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:${safeExtGet('kotlinVersion', '1.4.21')}"
+  implementation 'com.facebook.react:react-native:+'
```

Comment:
that is for `ReactNativeHostHandler` which need some react-native classes.
considering we are going to migrate as expo-modules, is it okay to add react-native dependency here?